docs/content/documentation/systems/Problem/contact/FrictionalContactProblem.md

```diff
@@ -1,8 +1,27 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # FrictionalContactProblem
 !syntax description /Problem/FrictionalContactProblem
 
+## Description
+
+`FrictionalContactProblem` is used when a user wants to use kinematic (default)
+enforcement of frictional contact. If a user wants to use the penalty method for
```

Comment:
@bwspenc should I mark this class as deprecated in the documentation system based on our rrp discussion?